packages/@o3r/eslint-config/schematics/ng-add/eslint/index.ts

```diff
@@ -1,50 +1,96 @@
+  workspaceProject.architect.lint ||= {
+    builder: '@angular-eslint/builder:lint',
+    options: {
+      eslintConfig: `${workspaceProject.root}/eslint.config.${extension}`,
+      lintFilePatterns: [
+        `${workspaceProject.sourceRoot || path.posix.join(workspaceProject.root, 'src')}/**/*.ts`
+      ]
+    }
+  };
+
+  workspace!.projects[projectName] = workspaceProject;
+  tree.overwrite('/angular.json', JSON.stringify(workspace, null, 2));
+};
+
 /**
  * Update ESLint Config
- * @param isWorkspace
  * @param rootPath
+ * @param projectName
  */
-export const updateEslintConfig = (isWorkspace = true, rootPath = __dirname): Rule => async (tree, context) => {
-  const { findFilesInTree, getTemplateFolder } = await import('@o3r/schematics');
-  const eslintConfigFiles = findFilesInTree(tree.root, (file) => /eslint.config.[cm]?js/.test(file));
+export const updateEslintConfig = (rootPath: string, projectName?: string): Rule => async (tree, context) => {
+  const { findFilesInTree, getTemplateFolder, getWorkspaceConfig } = await import('@o3r/schematics');
+  const workspace = getWorkspaceConfig(tree);
+  if (!workspace) {
+    return;
+  }
+  const workspaceProject = projectName ? workspace.projects[projectName] : undefined;
+  const projectRootPath = workspaceProject?.root || '.';
+  const workingDir = tree.getDir(projectRootPath);
+  const eslintConfigFiles = findFilesInTree(workingDir, (file) => /eslint.config.[cm]?js/.test(file));
   if (eslintConfigFiles.length > 1) {
     context.logger.warn(
       'Unable to add the "@o3r/eslint-config" recommendation because several ESLint config file detected.\n'
       + eslintConfigFiles.map((file) => `\t- ${file.path.toString()}`).join('\n')
     );
     return;
   }
+
   const templateOptions = {
     extension: 'mjs',
     codeBeforeConfig: '',
     codeAfterConfig: '',
     oldConfig: '',
-    packageName: (tree.readJson('package.json') as JsonObject).name,
-    detectedTsConfigs: findFilesInTree(tree.root, (f) => /tsconfig.*\.json/.test(f)).map((entry) => path.basename(entry.path)).concat('tsconfig.eslint.json')
+    relativePathToRoot: path.relative(projectRootPath, '.'),
+    packageName: (tree.readJson(workingDir.file(fragment('package.json'))?.path || 'package.json') as JsonObject).name,
+    detectedTsConfigs: findFilesInTree(workingDir, (f) => /tsconfig\..*\.json/.test(f)).map((entry) => path.basename(entry.path)).concat('tsconfig.eslint.json'),
+    isApp: workspaceProject?.projectType === 'application'
   };
+
   if (eslintConfigFiles.length === 1) {
     const file = eslintConfigFiles[0];
     const filePath = file.path.toString();
     const fileContent = file.content.toString();
     const extension = path.extname(filePath) as 'mjs' | 'cjs' | 'js';
-    const regexp = extension === 'mjs' ? /export\s+default\s+[^;]*;/ : /module.exportss+=\s+[^;]*;/;
+    const regexp = extension === 'mjs' ? /export\s+default\s+[^;]*;/ : /module.exports\s+=\s+[^;]*;/;
```

Comment:
```suggestion
    const regexp = extension === 'mjs' ? /export\s+default\s+[^;]*;/ : /module.exports\s*=\s*[^;]*;/;
```
Even if it does not look nice, `module.exports=something;` is valid no?